src/styles/variables.scss

```diff
@@ -0,0 +1,4 @@
+$main-accent: #060b35;
+$secound-accent: #00acdc;
+$third-accent: #616070;
```

Comment:
Rename variable names. Second, third is not informative at all. Variables should describe their purpose